tests/test_pytest/test_plugin_schema/good/products/bases/product.yaml

```diff
@@ -0,0 +1,29 @@
+---
+family: interpolator_algorithm_colormap
+name: infrared
+docstring: "Good product using family"
+spec:
+  source_name: abi
+  variables:
+    - B14BT
+  interpolator:
+    plugin:
+      name: pyresample.nearest_neighbor
+      arguments: {}
+  algorithm:
+    plugin:
+      name: single_channel
+      arguments: {}
+  colormap:
+    plugin:
+      name: matplotlib
+      arguments: {}
+
+---
+product_defaults: infrared
+name: infrared
```

Comment:
So did we land on requiring name and docstring in every product?

Reply:
We require `name` in every product, but `docstring` is required if using `family` and optional if using `product_defaults` because it can be pulled from `product_defaults`.